src/Microsoft.VisualStudio.Editors/PropPages/CodeAnalysisPropPage.vb

```diff
@@ -18,7 +18,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
         Private Const NuGetPackageManagerSearchProviderGuid As String = "042C2B4B-C7F7-49DB-B7A2-402EB8DC7892"
         Private Const RoslynAnalyzersDocumentationLink As String = "https://docs.microsoft.com/visualstudio/code-quality/roslyn-analyzers-overview"
 
-        Private Shared ReadOnly s_latestStableVersion As Version = New Version(2, 9, 3)
+        Private Shared ReadOnly s_latestStableVersion As Version = New Version(2, 9, 5)
```

Comment:
Is this going to need to be bumped all the time?

Reply:
Yes, unfortunately none of the NuGet APIs allow us to specify an upper bound. We cannot use the NuGet API that always installs the latest version because in future we are likely to release newer versions that require next VS release as the minimum version.

Reply:
That is tough, but even if they did, you wouldn't really know what to set as the upper bound.

We recently moved to using versions that match VS, ie our DLLs are now v16.4. I don't suppose you would do the same? Though that assumes there is an API for Nuget that would let you specify a base version and have it pick up the latest patch.

Reply:
We match the major and minor version of our analyzer packages with underlying Microsoft.CodeAnalysis on which the package is built. So, our 2.x packages need at least Microsoft.CodeAnalysis 2.x, 3.x packages need Microsoft.CodeAnalysis 3.x and so on. So, if such an API was provided, we could set 4.0 as an exclusive upper bound, so any latest package earlier then 4.0 should be installed.